kustomize/overlay/7.3.0/devops/configmap.yaml

```diff
@@ -8,3 +8,4 @@ data:
   MTLS_FQDN: mtls.sapig.devops.forgerock.financial
   IDENTITY_PLATFORM_FQDN: iam.devops.forgerock.financial
   OB_ASPSP_SOFTWARE_ID: gYcVCmFmzItBTxM4Y1cPn2
+  IG_OB_ASPSP_SIGNING_KID: KT85P5YZolPsbGVmHIOCm-KgRT0
```

Comment:
What does this have to do with the deprecated actions?

Reply:
ah looks like this change was outstanding, will sort